lib/orbit/import/import_certifications.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Orbit.Import.ImportCertifications do
+  alias Orbit.Certification
+  alias Orbit.Import.ImportHelpers
+  alias Orbit.Repo
+
+  @spec import!(Enumerable.t(), Certification.certification_type()) :: :ok
+  def import!(file_lines, certification_type) do
+    file_lines
+    |> Enum.drop_while(fn line -> String.trim(line) != "" end)
+    |> Enum.drop_while(fn line -> String.trim(line) == "" end)
+    |> ImportHelpers.parse_csv(true)
```

Comment:
Can `ImportHelpers.parse_csv` handle these empty line filters? I think we'd want them on all our imports. They might be arguments to the csv library, and if not, then they could be done in our `parse_csv`.

Reply:
These are actually explicitly for the LMS csvs that contain custom headers at the top—it's dropping those and the empty lines that precede the actual csv data. They aren't csv specific and should remain outside of `ImportHelpers`.